index.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express')
+const exphbs = require('express-handlebars')
+const MongoClient = require('mongodb').MongoClient
+const ObjectId = require('mongodb').ObjectId
+const cors = require('cors')
+const dateRange = require('./dateRange')
+
+const url = "mongodb://root:password@localhost:27017"
+
+const app = express()
+app.use(express.json())
+
+const port = 3000
+
+app.engine('handlebars', exphbs());
+app.set('view engine', 'handlebars');
+
+app.use(express.static('public'))
+
+app.use(cors())
+app.options('*', cors())
+
+app.get('/hotels', async (request, response) => {
+    const connection = await MongoClient.connect(url)
+    const db = connection.db('seal')
+    const collection = db.collection('hotels')
+    let start = new Date(request.query.checkin)
+    let end = new Date(request.query.checkout)
+    const data = await collection.find({'booked':
+            {$not:
+                    {$in:
+                            [
+                                start,
+                                end
+                            ]
+                    }
+            }}).toArray()
+    response.json(data)
+})
+
+app.post('/hotels/:id', async (request, response) => {
```

Comment:
if this is a REST API, then this route would create a new hotel, not a new booking. for new bookings it would be something like:
POST -> /booking?hotelId=254ytf5t536gc45